### DISCUSSION TITLE
How to have a silent lr_find()

### DISCUSSION THREAD

**Discussion by grudloff** [Lightning-AI/pytorch-lightning#9635] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Is there a way to make lr_find not print anything while searching?

**Comment by tchaton** [comment, marked as answer]
Hey @grudloff,

I don't believe this is supported. You could either capture the logs or contribute the feature :)

Best,
T.C

**Reply by grudloff** [reply]
Would be happy to contribute. I already silenced the progress bar but I am having a hard time pinpointing where the validation metrics are being logged like this:
```
--------------------------------------------------------------------------------
DATALOADER:0 VALIDATE RESULTS
{'val_MAE': 0.34362396597862244, 'val_loss': 0.16003160178661346}
--------------------------------------------------------------------------------
```
Could you give me a hand with this?
